src/Address.js

```diff
@@ -20,6 +21,20 @@ class Address {
     }
   }
 
+  // Converts address to RIPEMD-160 hash
+  toHash160(address) {
```

Comment:
This is cool. Can we update and add a couple things?

There are RIPEMD160 hashes and there are HASH160 hashes. A HASH160 is ` ripemd160(sha256())`. Do we know if these are normal RIPEMD160 or are they actually HASH160?

Can we wave

1. `toHash160` accepts either legacy or cashaddr and returns hash160 (you already have this)
2. `hash160ToLegacy` which returns legacy address. (you already have this. just need to rename)
3. `hash160ToCash` which returns cashaddr. (You basically already have this. just need to call `hash160ToLegacy` and convert to cashaddr before returning`

Also can you update [the `BITBOX.Address` docs](https://github.com/Bitcoin-com/developer.bitcoin.com/blob/master/html/pages/bitbox/docs/address.html) to include these new methods?

Thx